common/client-libs/validator-client/src/nymd/traits/vesting_execute_client.rs

```diff
@@ -0,0 +1,268 @@
+pub use crate::nymd::cosmwasm_client::signing_client::SigningCosmWasmClient;
+use crate::nymd::cosmwasm_client::types::ExecuteResult;
+use crate::nymd::error::NymdError;
+use crate::nymd::fee_helpers::Operation;
+use crate::nymd::{cosmwasm_coin_to_cosmos_coin, NymdClient};
+use async_trait::async_trait;
+use cosmwasm_std::Coin;
+use mixnet_contract::{Gateway, IdentityKey, MixNode};
+use vesting_contract::messages::ExecuteMsg as VestingExecuteMsg;
+
+#[async_trait]
+pub trait VestingExecuteClient {
+    async fn vesting_bond_gateway(
+        &self,
+        gateway: Gateway,
+        pledge: Coin,
+    ) -> Result<ExecuteResult, NymdError>;
+    async fn vesting_unbond_gateway(&self) -> Result<ExecuteResult, NymdError>;
```

Comment:
Can you maybe leave a line of space between multiple functions? I think it looks a bit cluttered